crates/aliveness/src/lib.rs

```diff
@@ -49,9 +48,9 @@ impl Aliveness {
             .wrap_err("failed to bind beacon socket")?;
 
         if let Some(ips) = ips {
-            Aliveness::send_beacons_unicast(&socket, ips).await?;
+            Self::send_beacons_unicast(&socket, ips).await?;
```

Comment:
Can't you drop the self here?

Reply:
The Aliveness::query function is also not associated with the type. Should I remove the struct Aliveness entirely or only keep Aliveness::query?

Reply:
Oh, I didn't even see that. Yes, I think it would be a good idea to remove it.

As it stands right now, all it does is being a glorified namespace, but since we're already in an "aliveness" module, I think that's redundant.